src/chat.rs

```diff
@@ -1980,6 +1981,21 @@ pub async fn send_msg_sync(context: &Context, chat_id: ChatId, msg: &mut Message
 }
 
 async fn send_msg_inner(context: &Context, chat_id: ChatId, msg: &mut Message) -> Result<MsgId> {
+    // protect all system messages againts LTRO attacks
```

Comment:
Why protect against attacks when _sending_ messages? Shouldn't this done when receiving?

Reply:
Ah, I think I understood, it's so that the security measures while receiving don't make problems during legitimate usage